mmcv/cnn/bricks/crf.py

```diff
@@ -0,0 +1,286 @@
+import torch
+import torch.nn as nn
+from typing import List, Optional
+
+
+class CRF(nn.Module):
+
```

Comment:
Add docstring here

Reply:
The docstring style is not correct, please refer to https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/models/backbones/resnet.py#L311-L386